app/javascript/controllers/activity_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="activity"
+export default class extends Controller {
+  static targets = ['activity', 'symbol']
+  connect() {
+  }
+
+  showDescription(event) {
+    const element = event.target
+    const index = element.dataset.index
+
+    if (this.activityTargets[index].style.display == "" || this.activityTargets[index].style.display == "none") {
+      this.activityTargets[index].style.display = "inline"
+      this.symbolTargets[index].classList.add("fa-caret-up")
+      this.symbolTargets[index].classList.remove("fa-caret-down")
+    } else if (this.activityTargets[index].style.display == "inline") {
+      this.activityTargets[index].style.display = "none"
+      this.symbolTargets[index].classList.add("fa-caret-down")
+      this.symbolTargets[index].classList.remove("fa-caret-up")
+    }
+  }
+}
```

Comment:
on click of an activity dropdown (only shown on laptop downward), activity will display. This is to maintain page 'neatness' and only show minimal and necessary information.